README.md

```diff
@@ -190,8 +190,8 @@ Feature discovery is preferably run as a Kubernetes DaemonSet. There is an
 example spec that can be used as a template, or, as is when just trying out the
 service:
 ```
-kubectl create -f rbac.yaml
-kubectl create -f node-feature-discovery-daemonset.json.template
+kubectl create -f https://raw.githubusercontent.com/kubernetes-incubator/node-feature-discovery/master/rbac.yaml
```

Comment:
Will the files be downloaded by default when cloning the repo/getting a release?

Reply:
Yes, they are part of the repo and will be downloaded. But, the catch here is that you don't need to clone anything to deploy the latest release